<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Settings</class>
 <widget class="QDialog" name="Settings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>526</width>
    <height>490</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(45, 45, 48);
font: 8pt &quot;Verdana&quot;;</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>362</x>
     <y>430</y>
     <width>131</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #004080;</string>
   </property>
   <property name="text">
    <string>Begin Simulation</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>400</y>
     <width>501</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;*Replacement Method is only available for &lt;span style=&quot; font-weight:600;&quot;&gt;Fully Associative &lt;/span&gt;cache type!&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>120</y>
     <width>361</width>
     <height>24</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="styleSheet">
       <string notr="true">color: rgb(112, 146, 190);</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;Memory Size&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="comboBox">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
selection-color: rgb(0, 0, 0);
selection-background-color: rgb(255, 255, 255);
color: #004080;</string>
      </property>
      <item>
       <property name="text">
        <string>1 MB</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2 MB</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>3 MB</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>4 MB</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>190</y>
     <width>361</width>
     <height>24</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="styleSheet">
       <string notr="true">color: rgb(112, 146, 190);</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Cache Size&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="comboBox_2">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
selection-color: rgb(0, 0, 0);
selection-background-color: rgb(255, 255, 255);
color: #004080;</string>
      </property>
      <item>
       <property name="text">
        <string>16 KB</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>8 KB</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>32 KB</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>260</y>
     <width>361</width>
     <height>24</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="styleSheet">
       <string notr="true">color: rgb(112, 146, 190);</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Cache Type&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="comboBox_3">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
selection-color: rgb(0, 0, 0);
selection-background-color: rgb(255, 255, 255);
color: #004080;</string>
      </property>
      <item>
       <property name="text">
        <string>Direct-Mapped</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Fully Associative</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>330</y>
     <width>361</width>
     <height>24</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QLabel" name="label_4">
      <property name="styleSheet">
       <string notr="true">color: rgb(112, 146, 190);</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Replacement Method&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="comboBox_4">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
selection-color: rgb(0, 0, 0);
selection-background-color: rgb(255, 255, 255);
color: #004080;</string>
      </property>
      <item>
       <property name="text">
        <string>Random Replacement</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>First In First Out</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Least Frequently Used</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
